monitoring/grafana/dashboards/.pylintrc

```diff
@@ -0,0 +1,541 @@
+[MASTER]
+
+# A comma-separated list of package or module names from where C extensions may
+# be loaded. Extensions are loading into the active Python interpreter and may
+# run arbitrary code
+# TODO: remove racially insensitive terms when this becomes fixed: https://github.com/PyCQA/pylint/issues/3669
+extension-pkg-whitelist=rados,rbd,math,cephfs
```

Comment:
I'm not a huge fan of big chunks of config (if an issue is found in one you'll have to remember fixing the same thing multiple times in different places... which is a mess). So why not pointing this pylint to a top-level dir config and (in a follow up PR, yes) pointing the dashboard (and the rest of pylint runs) to that one?

Reply:
linked to the dashboard one